build.gradle.kts

```diff
@@ -114,12 +112,12 @@ tasks {
         password = environment("PRIVATE_KEY_PASSWORD")
     }
 
-    publishPlugin {
-        dependsOn("patchChangelog")
-        token = environment("PUBLISH_TOKEN")
-        // The pluginVersion is based on the SemVer (https://semver.org) and supports pre-release labels, like 2.1.7-alpha.3
-        // Specify pre-release label to publish the plugin in a custom Release Channel automatically. Read more:
-        // https://plugins.jetbrains.com/docs/intellij/deployment.html#specifying-a-release-channel
-        channels = properties("pluginVersion").map { listOf(it.split('-').getOrElse(1) { "default" }.split('.').first()) }
-    }
+//    publishPlugin {
+//        dependsOn("patchChangelog")
+//        token = environment("PUBLISH_TOKEN")
+//        // The pluginVersion is based on the SemVer (https://semver.org) and supports pre-release labels, like 2.1.7-alpha.3
+//        // Specify pre-release label to publish the plugin in a custom Release Channel automatically. Read more:
+//        // https://plugins.jetbrains.com/docs/intellij/deployment.html#specifying-a-release-channel
+//        channels = properties("pluginVersion").map { listOf(it.split('-').getOrElse(1) { "default" }.split('.').first()) }
+//    }
```

Comment:
Is this block supposed to be removed?

Reply:
Yes, the plugin is not currently published to the marketplace